aspnetcore/performance/caching/response.md

```diff
@@ -63,6 +63,13 @@ The `ResponseCacheAttribute` is used to specify how a controller action's header
 
    When set, a `vary` response header will be written with the response.
 
+**VaryByQueryKeys `string[]`**
+
+   When set, the response caching middleware will vary the stored response by the given list of query keys. The middleware will serve the stored response only if the query keys of the incoming request matches those of the original request that generated the stored response.
+
+   > [!NOTE]
+   > There is no corresponding HTTP header for this property as it is a HTTP feature specifically handled by the response caching middleware. As such a response caching middleware must be configured on the server to set the `VaryByQueryKeys` property.
```

Comment:
Not sure if it's necessary to mention that setting this property without adding the middleware will throw an exception at runtime.